sdk/monitor/azure-monitor-query/README.md

```diff
@@ -0,0 +1,316 @@
+        row.getTableRow()
+            .forEach(cell -> System.out.println("Column = " + cell.getColumnName() + "; value = " + cell.getValueAsString()));
+    }
+}
+```
+
+### Create Metrics query client
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L56-L58 -->
+```java
+MetricsClient metricsClient = new MetricsClientBuilder()
+    .credential(tokenCredential)
+    .buildClient();
+```
+
+### Create Metrics query async client
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L60-L62 -->
+```java
+MetricsAsyncClient metricsAsyncClient = new MetricsClientBuilder()
+    .credential(tokenCredential)
+    .buildAsyncClient();
+```
+
+### Get metrics
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L161-L188 -->
+```java
+Response<MetricsQueryResult> metricsResponse = metricsClient
+    .queryMetricsWithResponse(
+        "{resource-id}",
+        Arrays.asList("SuccessfulCalls"),
+        new MetricsQueryOptions()
+            .setMetricsNamespace("Microsoft.CognitiveServices/accounts")
+            .setTimespan(Duration.ofDays(30).toString())
+            .setInterval(Duration.ofHours(1))
+            .setTop(100)
+            .setAggregation(Arrays.asList(AggregationType.AVERAGE, AggregationType.COUNT)),
+        Context.NONE);
+
+MetricsQueryResult metricsQueryResult = metricsResponse.getValue();
+List<Metrics> metrics = metricsQueryResult.getMetrics();
+metrics.stream()
+    .forEach(metric -> {
+        System.out.println(metric.getMetricsName());
+        System.out.println(metric.getId());
+        System.out.println(metric.getType());
+        System.out.println(metric.getUnit());
+        System.out.println(metric.getTimeSeries().size());
+        System.out.println(metric.getTimeSeries().get(0).getData().size());
+        metric.getTimeSeries()
+            .stream()
+            .flatMap(ts -> ts.getData().stream())
+            .forEach(mv -> System.out.println(mv.getTimeStamp().toString() + "; Count = " + mv.getCount()
+                + "; Average = " + mv.getAverage()));
+    });
+```
+
+## Key concepts
+
+### Logs
+
+Azure Monitor Logs is a feature of Azure Monitor that collects and organizes log and performance data from monitored
+resources. Data from different sources such as platform logs from Azure services, log and performance data from virtual
+machines agents, and usage and performance data from applications can be consolidated into a single workspace so they
+can be analyzed together using a sophisticated query language that's capable of quickly analyzing millions of records.
+You may perform a simple query that just retrieves a specific set of records or perform sophisticated data analysis to
+identify critical patterns in your monitoring data.
+
+#### Log Analytics workspaces
+
+Data collected by Azure Monitor Logs is stored in one or more Log Analytics workspaces. The workspace defines the
```

Comment:
Does this mean the azure resource (Monitor) may be in a different location and each of the workspaces could be in different locations?

Reply:
Yes, logs can be sent to different workspaces and queries can be written to get logs from more than one workspace.